lib/policies/serializationPolicy.ts

```diff
@@ -48,8 +48,8 @@ export class SerializationPolicy extends BaseRequestPolicy {
       const bodyMapper: Mapper | undefined = operationSpec.requestBodyMapper;
       if (bodyMapper) {
         try {
-          if (request.body != undefined) {
-            request.body = this._serializer.serialize(bodyMapper, request.body, "");
+          if (request.body != undefined && operationSpec.requestBodyName) {
```

Comment:
Make sure this covers scenarios where the request body is null. Basically someone wants to send null on the wire and it should be possible to do that.

Reply:
How is null different than an empty body? Do you mean that they want to send the text string "null"?

Reply:
It passes all of the tests in the autorest.typescript, which I'm guessing means it passes this request.

Reply:
I'm interpreting it to mean that down the line if someone wants to encode the body object `null` as JSON, then we should translate to the string `"null"` so that 4-character sequence will be written to the wire.

You might be able to do this just by changing `request.body != undefined` to `!==`. But I'm assuming you made some efforts to preserve the way the current serializer works. If we decide to handle this use case let's start by adding a failing test 👍